lib/util.js

```diff
@@ -15,11 +15,24 @@ function getPaths(publicPath, compiler, url) {
   const compilers = compiler && compiler.compilers;
   if (Array.isArray(compilers)) {
     let compilerPublicPath;
+
+    // the path portion of compilerPublicPath
+    let compilerPublicPathBase;
+
     for (let i = 0; i < compilers.length; i++) {
       compilerPublicPath = compilers[i].options
     && compilers[i].options.output
     && compilers[i].options.output.publicPath;
-      if (url.indexOf(compilerPublicPath) === 0) {
+
+      if (compilerPublicPath.startsWith('/')) {
```

Comment:
`startsWith` is convenient and semantic, but it's not as performant as `indexOf`. see: https://jsperf.com/startswith-vs-indexof-foo/1. I normally wouldn't nitpick on this, but being middleware we should take advantage of minor gains like this, and also be consistent.

Reply:
Done